src/index.js

```diff
@@ -0,0 +1,44 @@
+import Commenter from './commenter';
+
+const pastTense = new Map(
+  Object.entries({
+    create: 'created',
+    update: 'updated',
+  })
+);
+
+/**
+ * Using Travis environment variables to determine context, post a comment to the appropriate github
+ * pull request.
+ *
+ * @param {string} content GitHub flavored markdown to be displayed in the comment.
+ * @param {*} auth Any authorization string or object accepted by octokit.
+ * @param {string} purpose The purpose of the comment - multiple calls with the same purpose will
+ *   edit the existing comment on the same thread, if possible. Note that multiple comments may be
+ *   created, as we don't have thread-level content locking on GitHub.
+ */
+export default async function(content, { auth = null, purpose = null } = {}) {
```

Comment:
Really nice to come to the end of the PR here and see how everything fits together—very nicely factored! 😊